src/redprl/operator.sml

```diff
@@ -100,7 +100,7 @@ struct
    * MONO: the Standard ML built-in equality properly compares the operators.
    * POLY: we have to compare the operators manually. *)
   datatype mono_operator =
-   (* axioms *)
+   (* the only element in equality *)
```

Comment:
Yeah, I think `axiom` is confusing too. Maybe we could make the comment say something like `the trivial realizer`.